recipes/libopusenc/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "libopusenc" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.osuosl.org/pub/xiph/releases/opus/{{ name }}-{{ version }}.tar.gz
+  sha256: 8298db61a8d3d63e41c1a80705baa8ce9ff3f50452ea7ec1c19a564fe106cbb9
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkgconfig
+  host:
+    - libopus
+  run:
+    - libopus
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/opus/opusenc.h
+    - test -f ${PREFIX}/lib/libopusenc.a
```

Comment:
Please test the share shared lib  you can use `test -f ${PREFIX}/lib/libopusenc${SHLIB_EXT}` IIRC